integration_test/generate_expected_metrics/generate_expected_metrics.go

```diff
@@ -0,0 +1,274 @@
+//go:build integration_test
+
```

Comment:
Oh, I remembered a more useful reason why: executables have to be in a `main` package, so it'd conflict with any executables we might add in the future to `integration_test/`. Having executables in their own folders seems to be a better practice in that sense. Since it's the only executable for now then I can put it under `integration_test/` but with the caveat that we might have to move it in the future.